### PR TITLE
SQS implementation

### PR DESCRIPTION
**Pull request by AlessandroColucci**
Scaffold of the project and sqs producer

### CODE REVIEW

File: src/main/java/it/pagopa/interop/probing/scheduler/dto/PollingActiveEserviceResponse.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  private String id;
+
+  @JsonProperty("eserviceName")
+  private String eserviceName;
+
+  @JsonProperty("producerName")
+  private String producerName;
+
+  @JsonProperty("versionNumber")
+  private Integer versionNumber;
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/scheduler/dto/PollingActiveEserviceResponse.java:21]: are these information actually required for the polling?
```suggestion
```

I would expect only these fields:
- db record `id`, to update the record
- `technology`, to know how to call the service
- `basePath`, to know where to call the service

What are other fields used for?

File: .mvn/wrapper/maven-wrapper.jar
Comment (galales) [inline .mvn/wrapper/maven-wrapper.jar:1]: is this package necessary in the project? which feature does it offer?
All other projects don't seem to include this, I strongly suggest to use the same approach in every service

**Review by nttdata-rtorsoli — APPROVED** [review]

File: src/main/java/it/pagopa/interop/probing/scheduler/producer/ServicesSend.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  @Autowired
+  private ObjectMapper objectMapper;
+
+
+  public void sendMessage(PollingActiveEserviceResponse service) throws IOException {
+    SendMessageRequest sendMessageRequest = new SendMessageRequest().withQueueUrl(sqsUrl)
+        .withMessageBody(objectMapper.writeValueAsString(service));
+    amazonSQS.sendMessage(sendMessageRequest);
+    log.info("Service with id" + service.getId() + " with version " + service.getVersionNumber()
```
[the remark this answers is not in the document]
Comment (AlessandroColucci) [reply src/main/java/it/pagopa/interop/probing/scheduler/producer/ServicesSend.java:32]: Yes, i used it in the log just to recognize which service was being published

File: src/main/java/it/pagopa/interop/probing/scheduler/dto/PollingActiveEserviceResponse.java (hunk 1 above)
Comment (AlessandroColucci) [reply src/main/java/it/pagopa/interop/probing/scheduler/dto/PollingActiveEserviceResponse.java:21]: Ok, i will delete them. I just aligned my dto to the one of the first version of the api exposed by operations. I will inform them of the changes.

Comment (galales) [reply src/main/java/it/pagopa/interop/probing/scheduler/dto/PollingActiveEserviceResponse.java]: generally, I expect each service to use only the portion of data that it requires for its business logic

File: src/main/java/it/pagopa/interop/probing/scheduler/producer/ServicesSend.java [hunk 3]
```diff
@@ -29,7 +29,6 @@ public void sendMessage(PollingActiveEserviceResponse service) throws IOExceptio
     SendMessageRequest sendMessageRequest = new SendMessageRequest().withQueueUrl(sqsUrl)
         .withMessageBody(objectMapper.writeValueAsString(service));
     amazonSQS.sendMessage(sendMessageRequest);
-    log.info("Service with id" + service.getId() + " with version " + service.getVersionNumber()
-        + " has been published in SQS.");
+    log.info("Service with id" + service.getId() + " has been published in SQS.");
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/scheduler/producer/ServicesSend.java:32]: just a note: from the message it seems that the logged id is the `eserviceId`, while it is actually the id of the record in the table.
This difference should be clear in the message

**Review by galales — APPROVED** [review]

**Review by nttdata-rtorsoli — APPROVED** [review]

**Pull request merged**